wazero/wazero.go

```diff
@@ -0,0 +1,94 @@
+package main
```

Comment:
The folder is named `wazero`, from what I understand it is the way to allow `go install` to install a binary named `wazero`. It does mean `github.com/tetratelabs/wazero/wazero` though, stuttering but don't know if there's another way to support `go install` but may have missed it